source/_pages/privacy-policy.blade.php

```diff
@@ -0,0 +1,31 @@
+@extends('_layouts.main')
```

Comment:
I think that this file could be replaced by a header at markdown file. Like of all posts.